src/main/java/seedu/address/ui/UiPart.java

```diff
@@ -66,8 +67,8 @@ public T getRoot() {
     private void loadFxmlFile(URL location, T root) {
         requireNonNull(location);
         fxmlLoader.setLocation(location);
-        fxmlLoader.setController(this);
         fxmlLoader.setRoot(root);
+        fxmlLoader.setController(this);
```

Comment:
Same thing here, but I am guessing because of some dependencies you needed to setRoot(root) before you setController(this)

Reply:
yup